mac

```diff
@@ -166,29 +166,62 @@ brew_install_or_upgrade 'hub'
 # shellcheck disable=SC2016
 append_to_file "$HOME/.zshrc" 'eval "$(hub alias -s)"'
 
+
+fancy_echo 'Checking on Node.js installation...'
+
 if ! brew_is_installed "node"; then
-  if ! command -v n > /dev/null; then
-    fancy_echo 'Installing n and latest Node.js and NPM...'
-    curl -L http://git.io/n-install | bash -s -- -y latest
+  if command -v nvm > /dev/null; then
```

Comment:
I'm not sure why, but this check is returning true even if `nvm` does not exist, and thus the Node installation is skipped. When I run `if command -v nvm > /dev/null; then echo 'hi'; fi` directly in the command line, nothing is output, so I'm scratching my head as to why it doesn't work here.